_posts/2021-08-25-DWAngularMaterialDesignSolutions.md

```diff
@@ -0,0 +1,36 @@
+---
+title: "Utilising the Angular Material Library to Provide Bespoke Design Solutions"
+author: Daniel Warren
+categories:
+  - front-end-development
+  - ux-design
+tags:
+  - Angular
+  - UX
+  - GUI
+---
+
+
+The [Angular Material Library](https://material.angular.io/) is a collection of Material Design components which integrates with [Angular](https://angular.io/), a typescript based, open-source web development framework. The UI elements that the library provides are:
+
+- Versatile: the same elements can be used on a desktop or mobile browser.
+- Modifiable: elements can be adapted to suit unique design requirements.
+- Styleable: styles can be created and applied to all elements in a project, allowing for colour schemes to be lifted from a UX/UI Designer's prototype and applied during development.
+
+For these reasons Angular has become the foremost development framework within BGS Digital for application development, web development, and PWA (progressive web app) development.
+
+![Demo-Mobile](https://github.com/DanielLyleWilliamWarren/BlogDay_AngularMaterial/raw/main/assets/opacitysolution.gif)
```

Comment:
This has given me a headache! Can it be less flashy?

Reply:
Not sure there is really a straight forward solution for this, when demonstrating the UI in the browser in 'mobile mode' all elements that are interacted with are highlighted in blue. @Fuhji what are your thoughts, I personally don't find it too flashy but I spend far too much time looking at flashy things on computer screens... If the other .gifs in desktop browser mode aren't too flashy the UI could be demonstrated in that format.